### PR TITLE
improve afl++ build

### PR DESCRIPTION
**Pull request by vanhauser-thc**
new commit, copy more files (e.g. important llvm instrumentation .so files were missing)

### CODE REVIEW

**Comment by vanhauser-thc** [comment]
presubmit is complaining about things I haven't touched :)

File: infra/base-images/base-builder/Dockerfile [hunk 1]
```diff
@@ -135,7 +135,7 @@ WORKDIR $SRC
 RUN git clone -b stable https://github.com/google/AFL.git afl
 RUN git clone https://github.com/AFLplusplus/AFLplusplus.git aflplusplus && \
```
Comment (inferno-chromium) [inline infra/base-images/base-builder/Dockerfile:136]: Did you base this on certain branch, i don't see the code to add afl++ in current Dockerfile on master.

Comment (vanhauser-thc) [reply infra/base-images/base-builder/Dockerfile:136]: it is in the afl++ branch (also the target of this PR)

Comment (inferno-chromium) [reply infra/base-images/base-builder/Dockerfile:136]: ah, didnt know of that. Will let @jonathanmetzman review then.

**Comment by inferno-chromium** [comment]
This is done in another cl, closing

**Pull request closed without merging**